packages/jellyfish-transaction/src/tx.ts

```diff
@@ -56,6 +56,11 @@ export interface Vout {
   value: BigNumber // ------------------| 8 bytes
   script: Script // --------------------| n = VarUInt{1-9 bytes}, + n bytes
   tokenId: number // -------------------| 1 byte (Although it is VarUInt but disabled hence always 0x00)
+
+  // ISSUE(canonbrother): nValue same as value, nTokenId same as tokenId, its inconsistent vout struct issue
+  // https://github.com/DeFiCh/ain/blob/c812f0283a52840996659121a755a9f723be2392/src/masternodes/mn_checks.cpp#L441-L442
+  nValue?: BigNumber // ----------------| 8 bytes
```

Comment:
duplication here. c++ side only have `nValue, nTokenId` -> https://github.com/DeFiCh/ain/blob/948100ccc6d849c1e3d81e21d539ad9bb9d6f231/src/primitives/transaction.h#L140
we can either use `value,tokenId` or `nValue, nTokenId` since the variable name does not matter for serialized data. But latter would add more refactoring to jellyfish side existing code. suggesting to stick with `value,tokenId`

Reply:
hmm.. if stick back with `value, tokenId` it will never pass because this checker
https://github.com/DeFiCh/ain/blob/c812f0283a52840996659121a755a9f723be2392/src/masternodes/mn_checks.cpp#L441-L442

Reply:
should not be the case right? i mean in c++ side it constructs vout structure from serialized data. so it can use any name. should be ok as long as c++ side sticks to the same variables. correct me if i'm missing anything.

Reply:
it just a value, should not affected by diff property